cukinia

```diff
@@ -116,6 +127,12 @@ cukinia_http_request()
 	cukinia_runner _cukinia_http_request "$1"
 }
 
+# not: negate the issue of what follows
+not()
+{
+	__not=1 "$@"
```

Comment:
Personnaly, I would set

```
not() {
  ! "$@"
}
```

and keep both error flag `set -e` and test as is

```
if "$@" > /dev/null ; then
  _cukinia_result PASS
else
  _cukinia_result FAIL
  cukinia_failures=$((cukinia_failures + 1))
fi

Furthermore,this is out of scope but the shebang must be at least `/bin/ash` because we are using `local` variables.